core/imt/CMakeLists.txt

```diff
@@ -8,7 +8,7 @@ set(sources base.cxx TTaskGroup.cxx)
 if (imt)
   include_directories(SYSTEM ${TBB_INCLUDE_DIRS})
 
-  set(headers ${headers} ROOT/TPoolManager.hxx ROOT/TThreadExecutor.hxx ROOT/TFuture.hxx)
+  set(headers ${headers} ROOT/TThreadExecutor.hxx ROOT/TFuture.hxx)
```

Comment:
Has `TPoolManager.hxx` become internal to ROOT? This is just removing it from the dependencies, but I think you meant to not install it, maybe?

Reply:
It was in ROOT::Internal already, but should be in the /src directory instead of the /inc one. I moved it there :)